docs/content/usage-reporting.md

```diff
@@ -0,0 +1,217 @@
+| NGINX Management Suite 2.11 or later    | https://docs.nginx.com/nginx-management-suite                                                                         |
+| Docker 23 or later                      | https://docs.docker.com/get-docker/                                                                                   |
+| Kubernetes 1.22 or later                | Ensure you can access the Kubernetes API, with the ability to deploy a Kubernetes Deployment and a Kubernetes secret. |
+| kubectl 1.22 or later                   | https://kubernetes.io/docs/tasks/tools/#kubectl                                                                       |
+
+In addition to the software requirements, you must have the following:
+- Access to NGINX Management Suite with username and password for basic authentication, including the URL, username, and password, with the access to the `/api/platform/v1/k8s-usage` endpoints
+- Access to the Kubernetes cluster where the NGINX Ingress Controller is deployed, with the ability to deploy a Kubernetes Deployment and a Kubernetes Secret.
+- Access to public internet to pull the image, since the image is hosted in the NGINX container registry at `registry.nginx.com/cluster-connector`. Alternatively, you can pull the image down and push it to a private container registry that your cluster has access to.
+[//]: # (  TODO: Update the image and tag after published)
+
+## Setting up user accounts in NGINX Management Suite
+
+The NGINX Cluster Connector uses the NGINX Management Suite API to report the number of NGINX Ingress Controller nodes in the cluster. To use the NGINX Management Suite API, you must have a user account with the correct role.
+
+1. Create a role following the steps in [Create a Role](https://docs.nginx.com/nginx-management-suite/admin-guides/access-control/set-up-rbac/#create-role) section of the NGINX Management Suite documentation. Select these permissions in step 6 for the role:
+   - Module: Instance Manager
+   - Feature: Nginx Plus Usage
+   - Access: CRUD
+
+2. Create a user account following the steps in [Add Users](https://docs.nginx.com/nginx-management-suite/admin-guides/access-control/set-up-rbac/#add-users) section of the NGINX Management Suite documentation. In step 6, assign the user to the role created above. Note that currently only basic auth authentication is supported for usage reporting purposes.
+
+## Deploying the NGINX Cluster Connector
+
+1. Basic authentication is required to connect the NGINX Management Suite API. If you do not have a user account, create one following the above section . A base64 representation of the username and password is required to create the Kubernetes secret. In this example the username will be `foo` and the password will be `bar`. To obtain the base64 representation of a string, use the following command:
```

Comment:
The user account requirement should not need to be repeated.  It was the previous section.
You can simply say: "the username and password created in the previous section"

Both the username and password are stored in the cluster connector secret and need to be converted to base64

Reply:
I'm not sure exactly how to rewrite the whole paragraph but I tried to some changes.